### PR TITLE
Fix/add minor version nodejs

### PR DESCRIPTION
**Pull request by avpash43**
Added minor version nodejs and updated an existing role by coockiecutter #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,86 +2,52 @@ nodejs role
 =========
 [![License](https://img.shields.io/badge/license-Apache-green.svg?style=flat)](https://raw.githubusercontent.com/lean-delivery/ansible-role-nodejs/master/LICENSE)
 [![Build Status](https://travis-ci.org/lean-delivery/ansible-role-nodejs.svg?branch=master)](https://travis-ci.org/lean-delivery/ansible-role-nodejs)
+[![Build Status](https://gitlab.com/lean-delivery/ansible-role-nodejs/badges/master/build.svg)](https://gitlab.com/lean-delivery/ansible-role-nodejs)
 [![Galaxy](https://img.shields.io/badge/galaxy-lean__delivery.nodejs-blue.svg)](https://galaxy.ansible.com/lean_delivery/nodejs)
-![Ansible](https://img.shields.io/ansible/role/d/32264.svg)
-![Ansible](https://img.shields.io/badge/dynamic/json.svg?label=min_ansible_version&url=https%3A%2F%2Fgalaxy.ansible.com%2Fapi%2Fv1%2Froles%2F32264%2F&query=$.min_ansible_version)
+![Ansible](https://img.shields.io/ansible/role/d/role_id.svg)
```
Comment (pavelpikta) [inline README.md]: replace role_id to 32264

File: README.md [hunk 2]
```diff
@@ -2,86 +2,52 @@ nodejs role
...
+![Ansible](https://img.shields.io/badge/dynamic/json.svg?label=min_ansible_version&url=https%3A%2F%2Fgalaxy.ansible.com%2Fapi%2Fv1%2Froles%2Frole_id%2F&query=$.min_ansible_version)
 
-## Summary
+A brief description of the role goes here.
```
Comment (pavelpikta) [inline README.md]: restore previous readme content

File: meta/main.yml [hunk 3]
```diff
@@ -2,25 +2,11 @@
 galaxy_info:
   role_name: nodejs
   author: Lean Delivery Team <team@lean-delivery.com>
-  description: Node.js and global packages installation
+  description: Yet Another Ansible Role From Lean Delivery
```
Comment (pavelpikta) [inline meta/main.yml]: restore previous content

File: molecule/resources/tests/test_default.py [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
     os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
 
 
-@pytest.mark.parametrize('pkg', [
-  'nodejs'
-])
-def test_pkg(host, pkg):
-    package = host.package(pkg)
-    assert package.is_installed
+def test_hosts_file(host):
```
Comment (pavelpikta) [inline molecule/resources/tests/test_default.py]: restore previous python test

**Review by pavelpikta — DISMISSED** [review]

File: README.md [hunk 5]
```diff
@@ -2,17 +2,20 @@ nodejs role
 =========
 [![License](https://img.shields.io/badge/license-Apache-green.svg?style=flat)](https://raw.githubusercontent.com/lean-delivery/ansible-role-nodejs/master/LICENSE)
 [![Build Status](https://travis-ci.org/lean-delivery/ansible-role-nodejs.svg?branch=master)](https://travis-ci.org/lean-delivery/ansible-role-nodejs)
+[![Build Status](https://gitlab.com/lean-delivery/ansible-role-nodejs/badges/master/build.svg)](https://gitlab.com/lean-delivery/ansible-role-nodejs)
```
Comment (tgadiev) [inline README.md]:
```suggestion
[![Build Status](https://gitlab.com/lean-delivery/ansible-role-nodejs/badges/master/build.svg)](https://gitlab.com/lean-delivery/ansible-role-nodejs/pipelines)
```

File: .travis.yml [hunk 6]
```diff
@@ -10,7 +10,7 @@ services:
   - docker
 
 before_install:
-  - git clone -b ${lint_version} https://github.com/lean-delivery/ansible-lint-rules.git ~/ansible-lint-rules
+  - git clone https://github.com/lean-delivery/ansible-lint-rules.git ~/ansible-lint-rules
 
 install:
   - pip install --upgrade ansible==2.5.* docker-py molecule==2.17 pyOpenSSL PyYAML pytest==3.7.4
```
Comment (tgadiev) [inline .travis.yml]:
```suggestion
  - pip install --upgrade ansible==2.7.* docker-py molecule==2.20 pyOpenSSL PyYAML pytest
```

File: molecule/cloud-aws-direct/molecule.yml [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+      debian_family:
+        nodejs_version: 10*
+  playbooks:
+    create: ../resources/provisioning/AWS/create.yml
+    prepare: ../resources/provisioning/AWS/prepare.yml
+    destroy: ../resources/provisioning/AWS/destroy.yml
+  lint:
+    name: ansible-lint
+  env:
+    ANSIBLE_LIBRARY: ${ANSIBLE_LIBRARY}
```
Comment (tgadiev) [inline molecule/cloud-aws-direct/molecule.yml]:
```suggestion
```

File: molecule/cloud-aws-direct/molecule.yml [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+        nodejs_version: 10.15.3
+      debian_family:
+        nodejs_version: 10*
+  playbooks:
+    create: ../resources/provisioning/AWS/create.yml
+    prepare: ../resources/provisioning/AWS/prepare.yml
+    destroy: ../resources/provisioning/AWS/destroy.yml
+  lint:
+    name: ansible-lint
+  env:
```
Comment (tgadiev) [inline molecule/cloud-aws-direct/molecule.yml]:
```suggestion
```

File: tasks/system/Debian.yml [hunk 9]
```diff
@@ -21,7 +21,7 @@
     - name: Add node.js repositories
       apt_repository:
         repo: >-
-          deb https://deb.nodesource.com/node_{{ nodejs_version }}.x
+          deb https://deb.nodesource.com/node_{{ nodejs_major_version }}
```
Comment (tgadiev) [inline tasks/system/Debian.yml]: That should be moved to dedicated variable to be able to override it.

File: tasks/system/RedHat.yml [hunk 10]
```diff
@@ -18,8 +18,8 @@
 
     - name: Add nodesource repositories
       package:
-        name: "{{ nodejs_transport }}://rpm.nodesource.com/pub_{{ nodejs_version }}\
-              .x/el/{{ ansible_distribution_major_version }}/\
+        name: "{{ nodejs_transport }}://rpm.nodesource.com/pub_{{ nodejs_major_version }}\
```
Comment (tgadiev) [inline tasks/system/RedHat.yml]: That should be moved to dedicated variable to be able to override it.

**Review by tgadiev — CHANGES_REQUESTED** [review]

**Review by tgadiev — APPROVED** [review]

**Pull request merged**